### ISSUE TITLE
Fix sad loss curves and improve deep learning model

### ISSUE THREAD

**Issue by ashtanmistal** [ashtanmistal/Forest-Friends#12]
Testing accuracy peaks early in epochs; decreases while training accuracy rises. Current test accuracy caps at around 89.5%; train increases past this. Typically starts out at ~70 something - I estimate that's what the class balance is but will have to determine that further. Will have to do further hyperparameter tuning and model training. 

Unfortunately not all of this is fixable given the accuracy of the deep learning algorithm is bottlenecked by the accuracy of the clustering algorithm, and additionally by the fact that the training dataset is on UBC proper which has a different density of trees than, say, Pacific Spirit regional park where the majority of unclassified trees are. But this will provide a good baseline and difference in tree classes for the purpose of visualization.

**Comment by ashtanmistal** [comment]
will also have to look into whether a 0-65535 color value is allowed or if it needs to be truncated to 0-255 during preprocessing. If this gets changed, also convert from long to float in preprocessing as well (instead of just modifying the getter). will help with processing times and could help in accuracy if the color values are being capped or something.

**Comment by ashtanmistal** [comment]
> will also have to look into whether a 0-65535 color value is allowed or if it needs to be truncated to 0-255 during preprocessing. If this gets changed, also convert from long to float in preprocessing as well (instead of just modifying the getter). will help with processing times and could help in accuracy if the color values are being capped or something.

resolved

**Comment by ashtanmistal** [comment]
have not yet exceeded 89.5 via hyperparameter tuning so will use this model for the purposes of MinecraftUBC 3.0.
